src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -22,19 +33,23 @@ export const TodoFilter: React.FC = () => {
           type="text"
           className="input"
           placeholder="Search..."
+          value={inputQuery}
+          onChange={event => dispatch(actions.setQuery(event.target.value))}
```

Comment:
create handle function for it, sonething like that:
```
 const handleSearch = (e: ChangeEvent<HTMLInputElement>) => {
   dispatch(actions.setQuery(event.target.value));
 };
```
this function receive event by default.
create similar functions for all cases